### PR TITLE
roles/grafana: fix admin password check

### PR DESCRIPTION
**Pull request by pigay**
Looks like grafana-cli command needs `--homepath` to work properly.

### CODE REVIEW

**Comment by oxedions** [comment]
Dear @pigay 

Thanks a lot ! Legacy community role was maintained by @strus38 (role is currently on the way to be transferred to collections and supported by core team).
@strus38 Can you have a look ? 😊

File: roles/grafana/tasks/main.yml
Comment (strus38) [inline roles/grafana/tasks/main.yml:105]: Shouldn't this --homepath be a role variable? I fear it is not the same on all OSes.
Thanks

**Review by strus38 — CHANGES_REQUESTED** [review]

File: roles/grafana/tasks/main.yml
Comment (oxedions) [reply roles/grafana/tasks/main.yml:105]: Note: no need to add vars for all OS versions, just add it for the OS you need, I will complete it once its merged 😉

Comment (pigay) [reply roles/grafana/tasks/main.yml:105]: You're right.
@oxedions: shouldn't I provide a default location for all OSes to prevent failure on installations that didn't rely on this command to succeed?

Comment (pigay) [reply roles/grafana/tasks/main.yml]: I've put a variable in defaults/main.yml. Is it ok?

Comment (strus38) [reply roles/grafana/tasks/main.yml]: I would have liked them in:
https://github.com/bluebanquise/community/blob/main/roles/grafana/vars/[OS].yml
Could you do it? (sorry)

Comment (pigay) [reply roles/grafana/tasks/main.yml]: No problem ;)

**Comment by strus38** [comment]
You forgot to update the bars/main and bump the version
And also the readme.md to log your change.

**Review by strus38 — APPROVED** [review]

**Comment by strus38** [comment]
Ok since on a soon deprecated branch.

**Comment by oxedions** [comment]
Many thanks for this PR and fix.
I will now freeze this repository and archive it. All new PRs have to be made on the main bluebanquise repository. You will find all community roles into collections folders.
@pigay I will include your fix here into the bluebanquise repository. Do you agree I also add you into the related collection contributors ?

**Comment by pigay** [comment]
Thanks for merging.

I'll be happy to be added as a contributor ;)

Pierre

**Comment by oxedions** [comment]
Ok ;-)

**Comment by oxedions** [comment]
I merged your changes into main repository.

**Pull request merged**
